dot/parachain/backing/candidate_backing.go

```diff
@@ -221,7 +221,16 @@ func (cb *CandidateBacking) processOverseerMessage(msg any, chRelayParentAndComm
 	case CanSecondMessage:
 		cb.handleCanSecondMessage(msg)
 	case SecondMessage:
-		cb.handleSecondMessage()
+		err := cb.handleSecondMessage(msg.CandidateReceipt, msg.PersistedValidationData, msg.PoV, chRelayParentAndCommand)
+
+		if errors.Is(err, errWrongPVDForSecondingCandidate) ||
+			errors.Is(err, errUnknownRelayParentForSecondingCandidate) ||
+			errors.Is(err, errParaOutsideAssignmentForSeconding) ||
+			errors.Is(err, errAlreadySignedValidStatement) {
+			logger.Error(err.Error())
+			return nil
+		}
```

Comment:
Could you tell me what this check is for? 

Also, can we move this to a function?

Reply:
Also you can use a switch:

```go
switch err {
case errWrongPVDForSecondingCandidate, 
  errUnknownRelayParentForSecondingCandidate,
  ...:
    logger.Error(err.Error())
    return nil
}
```

Reply:
We may not need a function if it's using switch!

Reply:
> Could you tell me what this check is for?

In Rust, they are sometimes logging errors and returning the OK value instead of an error.
But I have returned an error from the function, and I am making sure we return nil error further for these particular errors.